src/tests/contract/utils/ApiRequestUtils.ts

```diff
@@ -0,0 +1,92 @@
+import { APIGatewayProxyEvent, APIGatewayProxyEventHeaders } from "aws-lambda";
+import { randomUUID } from "crypto";
+import { IncomingHttpHeaders } from "http";
+import { Constants } from "./Constants";
+
+export const eventRequest = {
+	body: "",
+	requestContext: {
+		requestId: randomUUID(),
+		accountId: "",
+		apiId: "",
+		authorizer: undefined,
+		protocol: "",
+		httpMethod: "",
+		identity: {
+			accessKey: "",
+			accountId: "",
+			apiKey: "",
+			apiKeyId: "",
+			caller: "",
+			clientCert: {
+				clientCertPem: "",
+				issuerDN: "",
+				serialNumber: "",
+				subjectDN: "",
+				validity: { notAfter: "", notBefore: "" },
+			},
+			cognitoAuthenticationProvider: "",
+			cognitoAuthenticationType: "",
+			cognitoIdentityId: "",
+			cognitoIdentityPoolId: "",
+			principalOrgId: "",
+			sourceIp: "",
+			user: "",
+			userAgent: "",
+			userArn: "",
+		},
+		path: "",
+		stage: "",
+		requestTimeEpoch: 0,
+		resourceId: "",
+		resourcePath: "",
+	},
+	headers: {},
+	multiValueHeaders: {},
+	httpMethod: "",
+	isBase64Encoded: false,
+	path: "",
+	pathParameters: null,
+	queryStringParameters: null,
+	multiValueQueryStringParameters: null,
+	stageVariables: null,
+	resource: "",
+};
+
+export function translateHttpRequestToApiGatewayProxyEvent(path: string, body: any, headers: IncomingHttpHeaders): APIGatewayProxyEvent {
```

Comment:
Wondering if this extra layer of abstraction maybe causes confusion? If we are using the route to figure out what function to call could we just call the right function from the route directly?

Reply:
refactored.